views_euid/privacy.hbs

```diff
@@ -6,137 +6,159 @@
+    
     <h2>Data retention</h2>
-    <p>We store each EUID for up to 30 days from the date on which the EUID was created. If you withdraw your consent, or object to our processing your data, we keep a record of your objection for 5 years in order to implement your request.</p>
-    <p>We keep a hashed copy of the email addresses and phone numbers used to create an EUID. This is necessary for us to ensure compliance with your data protection rights.</p>
+    <p>We may store your hashed email address or phone number for up to 18 months from the date on which the EUID was created in order to operate the EUID for certain advertisers and/or data partners. If you withdraw your consent, or object to our processing of your data for EUID, we keep a record of your opt-out, a hashed copy of the email address or phone number used to create the EUID, and the EUID itself. This is necessary for us to implement your request and ensure ongoing compliance with your data protection rights.</p>
+    
     <h2>Contact us</h2>
     <p>You may contact us regarding privacy matters as follows:</p>
-    <p>Email:<b>  privacy  </b>((at))<b>  thetradedesk  </b>((dot))<b>  com</b></p>
-    <p><strong>Global Data Protection Officer (DPO)</strong> The Trade Desk, Global Privacy Office 42 N. Chestnut St., Ventura, CA 93001 Email: <b>dpo</b> ((at)) <b>thetradedesk</b> ((dot)) <b>com</b></p>
+    <p>Email:<b>  privacy  </b>((at))<b>  thetradedesk  </b>((dot))<b>  com</b> or submit an inquiry <a href="https://privacyportal-eu.onetrust.com/webform/2abc1a63-35c5-4ef7-b11b-1c55714738b5/4c5ba598-b05c-4b1c-b3da-abab6ac1c9db">here</a></p>
+    <p><strong>Global Data Protection Officer (DPO)</strong> Email: <b>dpo</b> ((at)) <b>thetradedesk</b> ((dot)) <b>com</b></p>
     <p>If you have a concern regarding our privacy practices, please contact us via the contacts above. If after reasonable efforts, you believe your concern has not been satisfactorily addressed by us:</p>
     <ul>
       <li>In the EU, we recommend you contact the Data Protection Authority in the Member State where you normally reside, or where you work or where you believe a breach has occurred.</li>
+      <li>In Switzerland, we recommend you contact the Federal Data Protection and Information Commissioner (FDPIC).</li>
       <li>In the UK, we recommend you contact the Information Commissioner’s Office (ICO) of the United Kingdom.</li>
     </ul>
-    <h2>Changes to this policy</h2>
-    <p>We may revise this policy at any time. However, if we make material changes, we will not apply them retroactively.</p>
+
+    <h2>Changes to this notice</h2>
+    <p>We may revise this notice at any time. However, if we make material changes, we will not apply them retroactively.</p>
+
+    <h2>Last updated</h2>
+    <p>27 September 2024</p>
```

Comment:
This needs changing to today's date?

Reply:
I assume this date represents when the document was last updated by our legal team and not when it was updated on the site itself.  So I think this is the correct date.